cmd/mount-sys.go

```diff
@@ -197,8 +168,7 @@ func mountOverlayMounts(rootLabel string, dryRun bool) error {
 	}
 
 	overlays := []overlayMount{
-		{"/.system/etc", []string{"/.system/etc"}, "/var/lib/abroot/etc/" + rootLabel, "/var/lib/abroot/etc/" + rootLabel + "-work"},
-		{"/opt", []string{"/.system/opt"}, "/var/opt", "/var/opt-work"},
```

Comment:
/opt is missing now

Reply:
This has also been replaced by a symlink, as `/opt` is intended to be reserved for local admin and should not be used by the system (see: https://refspecs.linuxfoundation.org/FHS_3.0/fhs-3.0.html#optAddonApplicationSoftwarePackages).